specification/eventgrid/data-plane/Microsoft.ApiManagement/stable/2018-01-01/APIManagement.json

```diff
@@ -156,6 +156,116 @@
           "type": "string"
         }
       }
+    },
+    "ApiManagementSelfHostedGatewayCreatedEventData": {
```

Comment:
These event names are pretty long. Could we possibly remove "SelfHosted"?

Reply:
Not really as this event is solely aimed for our self-hosted gateway.

Here's a full list of all events:

- Microsoft.ApiManagement.SelfHostedGateway.Created
- 	Microsoft.ApiManagement.SelfHostedGateway.Updated
- 	Microsoft.ApiManagement.SelfHostedGateway.Deleted
- 	Microsoft.ApiManagement.SelfHostedGateway.HostnameConfiguration.Created
- 	Microsoft.ApiManagement.SelfHostedGateway.HostnameConfiguration.Updated
- 	Microsoft.ApiManagement.SelfHostedGateway.HostnameConfiguration.Deleted
- 	Microsoft.ApiManagement.SelfHostedGateway.CertificateAuthority.Created
- 	Microsoft.ApiManagement.SelfHostedGateway.CertificateAuthority.Updated
- 	Microsoft.ApiManagement.SelfHostedGateway.CertificateAuthority.Deleted
- 	Microsoft.ApiManagement.SelfHostedGateway.API.Added
- 	Microsoft.ApiManagement.SelfHostedGateway.API.Removed

Reply:
Do you anticipate there being similar events for the managed gateway at some point?

Reply:
I'd like to have this freedom, yet. Our ARM resource is big and these events are coming from https://learn.microsoft.com/en-us/azure/templates/microsoft.apimanagement/service/gateways/apis?pivots=deployment-language-bicep and related sub resources.

Our managed gateway also has custom hostname support as well as CAs.

Reply:
As part of our product we always make a distinction